component/install.php

```diff
@@ -570,7 +570,7 @@ private function updateTables() {
 		}
 		if (array_key_exists("extra_info", $cols))
```

Comment:
We discussed this on Glip. In case we face a very old migration that line is still needed.